youtube_dl/extractor/rutube.py

```diff
@@ -193,3 +193,59 @@ class RutubePersonIE(RutubeChannelIE):
     }]
 
     _PAGE_TEMPLATE = 'http://rutube.ru/api/video/person/%s/?page=%s&format=json'
+
+class RutubePlaylistIE(InfoExtractor):
+    IE_NAME = 'rutube:playlist'
+    IE_DESC = 'Rutube playlists'
+    _TESTS = [{
+        'url': 'https://rutube.ru/video/10b3a03fc01d5bbcc632a2f3514e8aab/?pl_id=4252&pl_type=source',
+        'info_dict': {
+            'id': '4252',
+        },
+        'playlist_count': 25,
+    }]
+
+    _VALID_URL = r'https?://rutube\.ru/(?:video|(?:play/)?embed)/(?P<id>[\da-z]{32})/.+pl_id=(?P<pl_id>\d+).*$'
```

Comment:
To be completely fair, I should use `suitable` here as well. The general issue is that rutube.ru playlists require two parameters on the web interface for playlists to initialise as intended, `pl_id` and `pl_type`. It may be wise to assume that the parameters can be in arbitrary order, however, I couldn't find out how to do that in pure regex. `.*$` was intended to indicate that `pl_type` may well come after `pl_id`.

I'll try to fix this using overloading `suitable`.